Dockerfile

```diff
@@ -3,6 +3,7 @@ FROM quay.io/assemblyline/nodejs:0.12.10
 RUN apt-get update -q\
```

Comment:
We use nodejs just to build frontend apps, it's not running anywhere on backend.